### ISSUE TITLE
LDA: all words have same probabilities in 8.2.1

### ISSUE THREAD

**Issue by barsunator** [JohnLangford/vowpal_wabbit#1134]
There seem to be an issue with LDA in 8.2.1. Looking at the readable model, all words have the same probability distribution for all topics, i.e.

Version 8.2.1
Id
Min label:0
Max label:0
bits:13
lda:10
0 ngram:
0 skip:
options: --lda 10 --lda_alpha 0.1 --lda_rho 0.1
Checksum: 1098308800
0 0.158531 0.110868 0.19577 55.8061 0.742481 45.7343 0.143204 0.281239 0.192346 0.759209 
1 0.158531 0.110868 0.19577 55.8061 0.742481 45.7343 0.143204 0.281239 0.192346 0.759209 
2 0.158531 0.110868 0.19577 55.8061 0.742481 45.7343 0.143204 0.281239 0.192346 0.759209 
3 0.158531 0.110868 0.19577 55.8061 0.742481 45.7343 0.143204 0.281239 0.192346 0.759209 
4 0.158531 0.110868 0.19577 55.8061 0.742481 45.7343 0.143204 0.281239 0.192346 0.759209 
5 0.158531 0.110868 0.19577 55.8061 0.742481 45.7343 0.143204 0.281239 0.192346 0.759209 
6 0.158531 0.110868 0.19577 55.8061 0.742481 45.7343 0.143204 0.281239 0.192346 0.759209 
...
and so on 

Trying the same example with 7.10 - all looks good, i.e.

Version 7.10.0
Min label:0.000000
Max label:1.000000
bits:13
0 pairs: 
0 triples: 
lda:10
0 ngram: 
0 skip: 
options: --lda 10
0 60.227139 0.171385 0.122895 1.363379 0.109052 0.117940 0.110771 64.940102 0.120219 0.105479 
1 0.102544 1.533017 257.000793 0.100472 0.101114 0.134646 0.101119 0.100732 0.100344 0.100362 
2 0.100340 0.100637 0.101130 90.852539 0.100305 0.101900 0.100037 0.100579 0.100170 0.102050 
3 0.100886 0.100389 0.101792 0.101506 0.101106 0.100455 0.100544 2030.231812 0.100883 0.100783 
4 5.876887 2.402472 4.066735 4.309469 251.200241 2.369248 1.398673 5.384004 4.604185 16.412397 
5 0.100253 0.100164 0.100269 0.100099 4759.302246 0.100199 0.100120 0.100200 0.100198 0.100150 
6 0.100122 0.100080 442.760468 0.100137 0.100197 0.100084 0.100023 0.100119 0.100028 0.100061 
...

this is reproducible in windows and linux built vw

command line used for training is:

vw -b 13 --lda 10 --lda_alpha 0.1 --lda_D=1057768 --lda_epsilon=0.1 --lda_rho=0.1  --readable_model model -d lda_input.vw 

training-time output doesn't seem to differ much between the 8.2.1 and 7.10

8.2.1
-----
finished run
number of examples = 1058327
weighted example sum = 1058327.000000
weighted label sum = 0.000000
average loss = 7.109359
total feature number = 18279588

7.10
-----
finished run
number of examples = 1058327
weighted example sum = 1.05833e+06
weighted label sum = 0
average loss = 7.14162
total feature number = 18279588

found another LDA related report, but it doesn't seem to have the same symptoms:
https://github.com/JohnLangford/vowpal_wabbit/issues/1096

**Comment by JohnLangford** [comment]
Is this a problem for the current master?

-John

**Comment by arielf** [comment]
Looks like the same issue as https://github.com/JohnLangford/vowpal_wabbit/issues/1119
(which seems fixed in latest).

**Comment by barsunator** [comment]
Indeed, this is same as 
https://github.com/JohnLangford/vowpal_wabbit/issues/1119

my mistake was i didn't remove is:open when searching :) apologies for the noise.
